ci/playbooks/edpm/run.yml

```diff
@@ -30,3 +30,8 @@
           {%-   endfor %}
           {%- endif %}
           -e @scenarios/centos-9/zuul_inventory.yml
+
```

Comment:
I know I pointed this file for this task, but after some more reflection, we should be a bit more generic.

The "flag" addition should happen elsewhere. I see two possibilities there:
either we add an actual playbook block in here:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/deploy-edpm.yml
for instance starting on Line 33, something like:
```YAML
- name: Inject status flag
  hosts: {{ cifmw_target_host | default('localhost') }}"
  tasks:
    - name: Inject success flag
      ansible.builtin.file:
        path: "{{ ansible_user_dir }}/cifmw-sucess
```

or maybe add the task in one of the sub-playbook - but not the "test" one since it may not be included...